File: Aula02/ex_2.5.c
#include <stdio.h>

int main() 
{
    char fila[3][20] = {"Ana", "Bruno", "Carla"};

    printf("Primeiro da fila: %s\n", fila[0]);

    printf("Cliente atendido: %s\n", fila[0]);

    printf("Fila atual:\n");
    printf("%s\n", fila[1]);
    printf("%s\n", fila[2]);

    return 0;
}

File: Aula02/ex_2.5.java
import java.util.LinkedList;
import java.util.Queue;

public class FilaSimples 
{
    public static void main(String[] args) 
    {
        Queue<String> fila = new LinkedList<>();

        fila.add("Ana");
        fila.add("Bruno");
        fila.add("Carla");

        System.out.println("Primeiro da fila: " + fila.peek());

        String atendido = fila.poll();
        System.out.println("Cliente atendido: " + atendido);

        System.out.println("Fila atual: " + fila);
    }
}